### DISCUSSION TITLE
Accessibility issue with Android version

### DISCUSSION THREAD

**Discussion by Pecant-Pie** [MCJack123/craftos2#245] [Q&A] [answered]
I'm not sure if there are certain hotkeys I'm unaware of within the file editor, but with the default android Keyboard I am unable to scroll up or down, and since I can't just click the down arrow key to move down a line, I get stuck on my first page of code and am unable to navigate further down. Is there some functionality I'm missing?

**Comment by MCJack123** [comment, marked as answer]
There are some issues with the Android version right now, and I haven't been able to work on them. You should be able to swipe down with two fingers to hit the down arrow key, but this is not working at the moment. I recommend you use a third-party keyboard like AnySoftKey until this issue is fixed.
